README.md

```diff
@@ -194,17 +194,19 @@ Dry run mode is only available within:
 You can import cloud-nuke into other projects and use it as a library for programmatically inspecting and counting resources. 
 
 ```golang 
+
 package main
 
 import (
 	"fmt"
 	"time"
 
+	"github.com/aws/aws-sdk-go/aws"
 	nuke_aws "github.com/gruntwork-io/cloud-nuke/aws"
+	nuke_creds "github.com/gruntwork-io/cloud-nuke/externalcreds"
```

Comment:
NIT: no need to alias this import since there isn't a conflicting name, like `aws` package.

Reply:
Good catch - fixed in 4318d48